files/ecr-credential-provider-config.json

```
@@ -9,7 +9,8 @@
         "*.dkr.ecr.*.amazonaws.com.cn",
         "*.dkr.ecr-fips.*.amazonaws.com",
         "*.dkr.ecr.us-iso-east-1.c2s.ic.gov",
-        "*.dkr.ecr.us-isob-east-1.sc2s.sgov.gov"
+        "*.dkr.ecr.us-isob-east-1.sc2s.sgov.gov",
+        "*.dkr.ecr.us-iso-west-1.c2s.ic.gov"
```

can't we do something like line 10? so we don't have to update this for every region in these partitions?
```suggestion
        "*.dkr.ecr.*.c2s.ic.gov"
```

---

Yes, fair callout. I have updated both the partition endpoints.